### DISCUSSION TITLE
New release?

### DISCUSSION THREAD

**Discussion by rusty-snake** [felixwiemuth/SimpleReminder#34] [Questions] [answered]
It looks like there where a lot of important changes under the hood done in the last month (targetSdk, SCHEDUELE_EXACT_ALARM, ...) but no release for 10 month.

Is there anything blocking? Or when do we can expect a new release?

**Comment by felixwiemuth** [comment, marked as answer]
Hi, thanks for your interest.

The next release is mainly a maintenance release (see #27). The app should generally work fine for now, and regarding that there were no reports of serious bugs, it wasn't necessary to quickly release this.

Now it shouldn't be long anymore until the release and you can also look forward to a few minor improvements and features :).

**Reply by felixwiemuth** [reply]
Update: It is out on F-Droid now.
